ballerina/Tests/tests.bal

```diff
@@ -0,0 +1,98 @@
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+isolated function  testCreateTranslation() returns io:Error? {
+    
+    CreateTranslationRequest payload = {
+        file: {
+            fileContent: content, fileName: "speech.mp3"},
+            model: "whisper-1"
+    };
+
+    CreateTranslationResponse|error result = openAIAudios->/audio/translations.post(payload);
+     
+    if result is error {
+        test:assertFail("Expected a successful response");
+             
+    } else {
+       test:assertEquals(result.text, "The");
+    }
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+isolated function  testCreateSpeech() returns error? {
+    // Define the payload for the request
+    CreateSpeechRequest requestPayload = {
+        model: "tts-1",
+        input: "The",
+        voice: "alloy",
+        response_format: "mp3",
+        speed: 1.0
+    };
+
+    // Send the POST request
+    byte[] response = check openAIAudios->/audio/speech.post(requestPayload);
+    
+    test:assertTrue(response.length() > 2000, msg = "Expected response length to be greater than 2000, but found " + response.length().toString());
+}
+
+
+@test:Config {
+    enable:false
+}
+isolated function  testCreateTranscription() returns error? {
+   
+   CreateTranscriptionRequest payload = {
+       file: {fileContent: content, fileName: "./speech.mp3"},
+       model: "whisper-1",
+       response_format: "verbose_json"  
+   };
+   
+    map<string|string[]> headers = {
+       "Authorization": "Bearer $OPENAI_API_KEY",
```

Comment:
No need to add this as the token will be automatically added to the client when we provide the `auth` configurations in the client initialization.
```suggestion
```